events-api/src/modules/events/events.controller.ts

```diff
@@ -21,5 +30,82 @@ EventsController.get(
       data: events,
       searchParams,
     });
-  }
+  },
+);
+
+EventsController.get(
+  "/:eventId/tickets",
+  validateEventId(eventsEventIdDtoSchema),
+  async (req, res) => {
+    const eventId = req.params["eventId"]!;
+
+    const tickets = await TicketsService.getTicketsByEventId(eventId);
+    if (!tickets) {
+      return res.status(404).json({
+        message: "Tickets not found",
+      });
+    }
+
+    return res.status(200).json({
+      message: "Tickets retrieved successfully",
+      data: tickets,
+    });
+  },
 );
+
+EventsController.post(
+  "/",
+  validateCreateEventsDto(createEventDtoSchema),
```

Comment:
Why to have special a create event DTO middleware? Why not just use the general one?

Reply:
> Why to have special a create event DTO middleware? Why not just use the general one?

you meant `validateSearchParams` ? if so should I write a general schema validator like this one